HISTORY.rst

```diff
@@ -4,6 +4,14 @@ History
 
 .. current developments
 
+current version
+---------------
+
+Compatibility
+~~~~~~~~~~~~~
+
+* Drop ``matplotlib.font_manager._rebuild()`` call in ``design.__init__`` - no longer supported (:issue:`96`)
+
```

Comment:
v1.1.1 never actually built b/c of this CI issue, so should we add this under v1.1.1 instead?